includes/widgets/google-maps.php

```diff
@@ -112,8 +112,9 @@ protected function register_controls() {
 					'api_key_notification',
 					[
 						'type' => Controls_Manager::RAW_HTML,
+						/* translators: %1$s Integration settings link open tag, %2$s: Link close tag, %3$s Create API key link open tag, %4$s: Link close tag. */
```

Comment:
```suggestion
						/* translators: %1$s Integration settings link open tag, %2$s: Link close tag, %3$s Create API key link open tag. */
```

You can use the close tag (the second parameter) also for closing the second link. please remove the 4th parameter (line 121)